go/vt/vttablet/grpctabletconn/conn.go

```diff
@@ -40,15 +41,29 @@ import (
 const protocolName = "grpc"
 
 var (
-	cert = flag.String("tablet_grpc_cert", "", "the cert to use to connect")
-	key  = flag.String("tablet_grpc_key", "", "the key to use to connect")
-	ca   = flag.String("tablet_grpc_ca", "", "the server ca to use to validate servers when connecting")
-	crl  = flag.String("tablet_grpc_crl", "", "the server crl to use to validate server certificates when connecting")
-	name = flag.String("tablet_grpc_server_name", "", "the server name to use to validate server certificate")
+	cert string
+	key  string
+	ca   string
+	crl  string
+	name string
 )
 
+func registerFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&cert, "tablet_grpc_cert", cert, "the cert to use to connect")
+	fs.StringVar(&key, "tablet_grpc_key", key, "the key to use to connect")
+	fs.StringVar(&ca, "tablet_grpc_ca", ca, "the server ca to use to validate servers when connecting")
+	fs.StringVar(&crl, "tablet_grpc_crl", crl, "the server crl to use to validate server certificates when connecting")
+	fs.StringVar(&name, "tablet_grpc_server_name", name, "the server name to use to validate server certificate")
+}
```

Comment:
I am assuming these are the flags that are required on the binaries that have the grpc server side code

Reply:
no, these are the flags that people that want to dial a `tabletconn` (client -> returns a `queryservice.QueryService`) need